### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Davy-G**

### CODE REVIEW

File: src/Application/Interfaces/IAppDbContext.cs [hunk 1]
```diff
@@ -4,10 +4,5 @@ namespace Application.Interfaces;
 
 public interface IAppDbContext
```
Comment (ddjerqq) [inline src/Application/Interfaces/IAppDbContext.cs:5]: you will need SaveChangesAsync eventually

File: src/Application/Interfaces/IMediator.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+namespace Application.Interfaces;
+
+public interface IMediator
```
Comment (ddjerqq) [inline src/Application/Interfaces/IMediator.cs:3]: you should add a reference to the Mediator package to the Application layer, and use Mediator.IMediator

Comment (Davy-G) [reply src/Application/Interfaces/IMediator.cs:3]: should i use abstraction

Comment (ddjerqq) [reply src/Application/Interfaces/IMediator.cs:3]: no, use the `Mediator.IMediator` interface

**Pull request merged**